### PR TITLE
dhcrelay: change the description and help text to make it clear multiple servers are supported

### PR DESCRIPTION
**Pull request by GurliGebis**
The wording on the dhcp relay page is not making it clear that multiple servers are support, unless you click the help button.

This fixes it, and changes the help text to make multiple servers the default explaination.

### CODE REVIEW

File: src/www/services_dhcp_relay.php [hunk 1]
```diff
@@ -160,11 +160,11 @@
                       </td>
                     </tr>
                     <tr>
-                      <td><a id="help_for_server" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("Destination server");?></td>
+                      <td><a id="help_for_server" href="#" class="showhelp"><i class="fa fa-info-circle"></i></a> <?=gettext("Destination server(s)");?></td>
```
Comment (fichtner) [inline src/www/services_dhcp_relay.php]: parenthesis get in the way, please use "Destination Servers"

File: src/www/services_dhcp_relay.php [hunk 2]
```diff
@@ -160,11 +160,11 @@
...
                       <td>
                         <input name="server" type="text" value="<?=!empty($pconfig['server']) ? htmlspecialchars($pconfig['server']):"";?>" />
                         <div class="hidden" for="help_for_server">
-                          <?=gettext("This is the IP address of the server to which DHCP requests are relayed. You can enter multiple server IP addresses, separated by commas. Select \"Proxy requests to DHCP server on WAN subnet\" to relay DHCP packets to the server that was used on the WAN interface.");?>
+                          <?=gettext("This is the IP addresses of the servers to which DHCP requests are relayed. You can enter multiple server IP addresses, separated by commas. Select \"Proxy requests to DHCP server on WAN subnet\" to relay DHCP packets to the server that was used on the WAN interface.");?>
```
Comment (fichtner) [inline src/www/services_dhcp_relay.php]: "These are the IP addresses of servers"

**Comment by GurliGebis** [comment]
Better? :)

**Comment by fichtner** [comment]
Merged, many thanks! :)

**Comment by GurliGebis** [comment]
Any time :)

**Pull request merged**